components/ILIAS/Forum/classes/GUI/class.ilForumThreadFormGUI.php

```diff
@@ -97,6 +93,7 @@ private function addMessageInput(): void
             'pastetext',
             'formatselect'
         ]);
+        $message->removePlugin('ilimgupload');
```

Comment:
If https://github.com/ILIAS-eLearning/ILIAS/pull/8091 got merged, we don't need this anymore.